### PR TITLE
Added solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,41 @@
...
+        super().__init__(name, appetite=3, is_hungry=is_hungry)
+
+    def catch_mouse(self) -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, is_hungry)
+        self.appetite = 7
```
Comment (pavlejviki) [inline app/main.py:34]: you can pass the value for self.appetite straight to super().__init__ in the same way as in class Cat

**Review by pavlejviki — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1 +1,40 @@
...
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite=3, is_hungry=is_hungry)
+
+    def catch_mouse(self) -> None:
+        print("The hunt began!")
```
Comment (viktoria-rybenchuk) [inline app/main.py:28]: Why do you need 'self' here? This method is static. Read checklist

File: app/main.py [hunk 3]
```diff
@@ -1 +1,40 @@
...
+
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite=7, is_hungry=is_hungry)
+
+    def bring_slippers(self) -> None:
+        print("The slippers delivered!")
```
Comment (viktoria-rybenchuk) [inline app/main.py:36]: According to the comment above, why do you need 'self' here?

File: app/main.py [hunk 4]
```diff
@@ -1 +1,40 @@
...
+
+
+def feed_animals(animals: list) -> int:
```
Comment (viktoria-rybenchuk) [inline app/main.py:39]: specify type annotation for animals

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by viktoria-rybenchuk — APPROVED** [review]
Good job!

**Pull request closed without merging**